add-dynamic-entity-layer/src/main/java/com/esri/arcgismaps/sample/adddynamicentitylayer/components/BottomSheetContent.kt

```diff
@@ -0,0 +1,182 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.adddynamicentitylayer.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleTypography
+
+/**
+ * Composable component to display Dynamic Entity Layer Settings
+ */
+@Composable
+fun DynamicEntityLayerProperties(
+    onTrackLineVisibilityChanged: (Boolean) -> Unit = { },
+    onPrevObservationsVisibilityChanged: (Boolean) -> Unit = { },
+    onObservationsChanged: (Float) -> Unit = { },
+    onPurgeAllObservations: () -> Unit = { },
+    isTrackLineVisible: Boolean,
+    isPrevObservationsVisible: Boolean,
+    observationsPerTrack: Float,
+    onDismiss: () -> Unit = { }
+) {
+    var sliderValue by remember { mutableStateOf(observationsPerTrack) }
```

Comment:
You could remove this as well, and use `observationsPerTrack` to set the slider value.

Reply:
missed removing this earlier. Thanks!